static/schemas/2.0/user.2.0.json

```diff
@@ -0,0 +1,30 @@
+{
+    "title": "User.2.0",
+    "definitions": {
+        "PartialUser": {
+            "description": "A RackHD local user",
+            "type": "object",
+            "properties": {
+                "username": {
+                    "description": "The username of the user",
+                    "type": "string"
+                },
+                "password": {
+                    "description": "The password of the user",
+                    "type": "string"
+                },
+                "role": {
```

Comment:
@dalebremner @yyscamper @keedya Long term plan is to allow roles to be added, removed or modified, so the set of enumerated values could change at run-time.  "Read", "Write", etc are privileges not roles and it makes sense to enumerate them in the schema for the roles route when it is available.